stats/opentelemetry/opentelemetry.go

```diff
@@ -171,6 +176,14 @@ func removeLeadingSlash(mn string) string {
 	return strings.TrimLeft(mn, "/")
 }
 
+func isMetricsDisabled(mo MetricsOptions) bool {
+	return mo.MeterProvider == nil
+}
+
+func isTracingDisabled(to experimental.TraceOptions) bool {
```

Comment:
We should discuss options, but requiring these to be called multiple times from the stats handler feels problematic.  Maybe there's a tracing stats handler and a metrics stats handler and we combine them based on which ones are enabled?

At the very least, these should become methods so we don't have to keep reaching into our data structures when calling them to find the options fields.

Reply:
@dfawley I have updated the PR with the later approach, as it makes sense to me for this to be method of Options struct. Let me know if you think this can be improved.